content/blog/2020-08-27-august-20-community-gems.md

```diff
@@ -0,0 +1,179 @@
+---
+title: August '20 Community Gems
+date: 2020-08-27
+description: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  using CI/CD to validate models, advanced DVC pipeline scenarios, and how CML adds pictures
+  to your GitHub and GitLab comments.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  using CI/CD to validate models, advanced DVC pipeline scenarios, and how CML adds pictures
+  to your GitHub and GitLab comments.
+picture: 2020-08-27/Gems_Aug_20.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/august-20-community-gems/477
+tags:
+  - Discord
+  - Gems
+  - CML
+  - hyperparameters
+  - LFS
+  - pipeline
+---
+
+Here are some of our top Q&A's from around the community. With the launch of
+[CML](https://cml.dev) earlier in the month, we've got some new ground to cover!
+
+## DVC questions
+
+### [Q: What's the relationship between your DVC remote and cache? If I have an external cache, do I really need a DVC remote](https://discordapp.com/channels/485586884165107732/563406153334128681/747588572479094866)
+
+You can think of your DVC remote similar to your Git remote- it's a place to
+backup and share artifacts, like datasets and models, and it gives you methods
+to push and pull those artifacts to and from your team.
+
+By default, `.dvc/cache` is also similar to your `.git` cache. They're both
+_local_ caches that optimize access to files (data and models for DVC cache, and
+code for Git cache).
+
+Your DVC remote is a superset of `.dvc/cache`- everything in your cache is a
+copy of something in your remote (though there may be files in your DVC remote
```

Comment:
Usually, your DVC remote ...

files that are not in your cache and wise versa ...

Reply:
added